components/global/core/io.cellery.observability.api/src/main/java/io/cellery/observability/api/registeration/IdpClientApp.java

```diff
@@ -143,5 +175,20 @@ public void checkServerTrusted(java.security.cert.X509Certificate[] certs, Strin
         return builder.build();
 
     }
-}
 
+    private static JSONObject checkRegisteration(String dcrEp, HttpClient client) {
+        try {
+            HttpGet getRequest = new HttpGet(dcrEp + Constants.CLIENT_NAME_PARAM + Constants.APPLICATION_NAME);
+            getRequest.setHeader(Constants.AUTHORIZATION, basicAuthentication());
+
+            HttpResponse resp = client.execute(getRequest);
+            HttpEntity entity = resp.getEntity();
+            String result = EntityUtils.toString(entity, Charset.forName("utf-8"));
+            return new JSONObject(result);
+        } catch (IOException | ConfigurationException e) {
+            log.error("Error occured while registering new Client in IDP" + e);
```

Comment:
instead of + , let's use ,

Reply:
Is this message correct ? Isn't this checking whether the client is already registered or not ?. Is this registrering the client ?